front-end/src/api/admin.js

```diff
@@ -31,3 +31,31 @@ export function saveUserPermissions(id, roles) {
     data: { roles },
   });
 }
+
+export function getUsersToApprove() {
+  return request({
+    url: BASE_API_URL + LMS_URLS.approve.approve,
+    method: "get",
+  });
+}
+
+export function approveUser(id) {
+  return request({
+    url: BASE_API_URL + LMS_URLS.approve.activate + `${id}`,
+    method: "post",
+  });
+}
+
+export function putUserOnWait(id) {
+  return request({
+    url: BASE_API_URL + LMS_URLS.approve.await + `${id}`,
+    method: "post",
+  });
+}
+
+export function disapproveUser(id) {
+  return request({
+    url: BASE_API_URL + LMS_URLS.approve.decline + `${id}`,
+    method: "post",
+  });
```

Comment:
The trailing slash is missing; Django will respond with 302.

Reply:
Added.